cal/cal/models.py

```diff
@@ -584,6 +584,21 @@ def query(self, calendar_ids=None, start=None, end=None):
         return events_qs
 
 
+    def queryWeek(self, calendar_ids=None, start=None, end=None):
+        queryset = []
```

Comment:
Changed it to get_hours_per_week because it doesn't get the events per week, but gets the hours per week instead.